src/store/reducers/positions.ts

```diff
@@ -171,6 +173,10 @@ const positionsSlice = createSlice({
     resetState(state) {
       state = defaultState
       return state
+    },
+    setShouldNotUpdateRange(state, action: PayloadAction<boolean>) {
```

Comment:
Why did you choose this solution? Is there a place where updating tick indexes on the chart is needed?